### ISSUE TITLE
Add a small (tiny) text at the beginning of each module

### ISSUE THREAD

**Issue by GaelVaroquaux** [INRIA/scikit-learn-mooc#115]
This text should:

* Give in plain English what the module is about
* Give the required skills for the module
* Point to resources to learning these skills
* Give the learning objectives
* Give the investment in time

**Comment by lfarhi** [comment]
Give also Objectives and Investment time

**Comment by GaelVaroquaux** [comment]
@lfarhi : can you modify my list above to add it. Thanks!!
